addons/mail/static/src/new/discuss/sidebar.xml

```diff
@@ -107,6 +107,7 @@
     >
         <div class="position-relative d-flex ms-4" style="width:30px;height:30px">
             <img class="w-100 h-100 rounded-circle" t-att-src="thread.imgUrl" alt="Thread Image"/>
+            <PartnerImStatus t-if="thread.type === 'chat'" className="'o-mail-category-im-status'" hasPositionAbsolute="true" partner="messaging.partners[thread.chatPartnerId]"/>
```

Comment:
Instead of introducing `hasPositionAbsolute` prop, it's best to give the classnames in `className` prop.

That way, no need to define another prop, and the style customisation of this component is apparent in this template. This is desirable so that we properly see `position-relative` and `position-absolute` at the same place.